reference-architecture/aws-ansible/playbooks/roles/etcd-volume/tasks/main.yaml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Create filesystem for /var/lib/etcd
+  filesystem:
+      fstype: xfs
+      dev: /dev/xvdc
+
+- name: Create local volumes directory
+  file:
+      path: /var/lib/etcd
+      state: directory
+      recurse: yes
+
+- name: Create fstab entry
+  mount:
+      name: /var/lib/etcd
+      src: '/dev/xvdc'
+      fstype: xfs
+      state: present
+
+- name: Mount fstab entry
+  mount:
+      name: /var/lib/etcd
+      src: '/dev/xvdc'
+      fstype: xfs
+      state: mounted
```

Comment:
I think we might skip cloud_init until a future release of cloud-init. I've spent a chunk of the morning comparing our code with demo-ansible and even testing the code from demo-ansible. When running cloud-init I'm consistently hitting the following error when trying to partition the filesystem.  

`Sep  9 11:05:58 localhost cloud-init: Cloud-init v. 0.7.6 running 'modules:config' at Fri, 09 Sep 2016 15:05:58 +0000. Up 65.76 seconds.
Sep  9 11:05:58 localhost cloud-init: 2016-09-09 11:05:58,949 - stages.py[WARNING]: Could not find module named cc_fs_setup (searched ['cc_fs_setup', 'cloudinit.config.cc_fs_setup'])`

I'm not sure if you have any ideas on this. Right now to get cloud-init to do the needful its a series of runcmd's which seems clunky.